src/App.tsx

```diff
@@ -15,49 +15,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_ALPHABET = 'Alphabet';
+const SORT_FIELD_LENGTH = 'Length';
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, reverse }: { sortField: string, reverse: boolean },
+) : string[] {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD_ALPHABET:
+          return good1.localeCompare(good2);
+
+        case SORT_FIELD_LENGTH:
+          return good1.length - (good2.length);
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reverse) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
+  const [reverse, setReverse] = useState(false);
```

Comment:
```suggestion
  const [isReverse, setIsReverse] = useState(false);
```
Prefix boolean variables with `is`